packages/core/data-transfer/lib/providers/local-strapi-destination-provider/index.ts

```diff
@@ -95,4 +95,30 @@ class LocalStrapiDestinationProvider implements IDestinationProvider {
 
     return mapSchemasValues(schemas);
   }
+
+  async getConfigurationStream(): Promise<Writable> {
+    if (!this.strapi) {
+      throw new Error('Not able to stream Configurations. Strapi instance not found');
+    }
+
+    const strapi = this.strapi;
```

Comment:
You don't need this if you use

```js
write: async (config, _encoding, callback) => {
```

on line `108`

and then

```js
await restoreConfigs(this.strapi, config);
```

on line `110`